AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/CallingContext.java

```diff
@@ -220,10 +224,11 @@ public Boolean isTranscriptionActive() {
         });
     }
 
-    public CompletableFuture hangupAsync() {
+    public CompletableFuture<Void> hangupAsync() {
```

Comment:
What does adding the <Void> do?

Reply:
adding `<Void>` resolves a Java compiler warning about `CompletableFuture` not having a parameterized type